packaging/python/python_packager.sh.in

```diff
@@ -1,24 +1,70 @@
-#!/bin/bash -xe
+#!/bin/bash -e
 
-for i in $*; do
-	# Set some plugin dirs
-	if [[ $i = *"musicPlugins"* ]]; then
-		musicPythonInterpreterDir="$i/medPipeline/python/interpreter"
+pwd
+
+cd @PROJECT_BINARY_DIR@
+mkdir -p python_interpreter
+
+MUSIC_LEARNING_DIR=@PROJECT_BINARY_DIR@/../../music-learning
+
+if [ ! -z "$MUSIC_LEARNING_DIR" ]; then
+	cd $MUSIC_LEARNING_DIR
+	zip -r ventriclesegmentation.zip ventriclesegmentation
+	if [ "$?" != "0" ]; then
+		printf "[Warning] unable to compress ventriclesegmentation repository. It will be not available in package" 
+	else
+		mv ventriclesegmentation.zip @PROJECT_BINARY_DIR@/python_interpreter/.
+	fi
+	zip -r afct.zip afct
+	if [ "$?" != "0" ]; then
+		printf "[Warning] unable to compress afct repository. It will be not available in package" 
+	else
+		mv afct.zip @PROJECT_BINARY_DIR@/python_interpreter/.
 	fi
-done
-
-# copy python interpreter (the compressed one) 
-if [ ! -f $musicPythonInterpreterDir/music-py27.tar.gz ]; then
-
-	virtualenv --version
-	cd $musicPythonInterpreterDir
-	virtualenv -p /usr/bin/python2.7 music-py27
-	source music-py27/bin/activate
-	pip install -r requirements.txt
-	deactivate
-	tar -zcvf music-py27.tar.gz music-py27 
+else
+	echo "music-learning directory does not exists."
 fi
 
 cd @PROJECT_BINARY_DIR@
-mkdir -p python_interpreter
-cp -p $musicPythonInterpreterDir/music-py27.tar.gz python_interpreter/.
+
+VTK_DIR=@PROJECT_BINARY_DIR@/../../python-packages/vtk-5.10.1/
+VTK_BIN_DIR=$VTK_DIR/bin
+OSTYPE="`uname`"
+
+if [ ! -z "$VTK_DIR" ]; then
+	if [ "$OSTYPE" = "Darwin" ]; then
```

Comment:
So, python interpreter is only enable on mac, is it correct?

Reply:
No. We must Change dynamic shared library install names for machos only.
But vtk python module is embedded for linux too